index.bs

```diff
@@ -1880,14 +1886,14 @@ Charter Extension</h3>
 	and <a href="#invited-expert-wg">Invited Experts</a>.
 
 	Advisory Committee representatives <em class="rfc2119">may</em> initiate
-	an <a href="#ACAppeal">Advisory Committee Appeal</a> against a Director's decision
+	an <a href="#ACAppeal">Advisory Committee Appeal</a> against a [=Team decision=]
 	regarding the extension of a Working Group or Interest Group charter.
 
 <h3 id="GeneralTermination">
 Chartered Group Closure</h3>
 
 	A [=Working Group=] or [=Interest Group=] charter specifies a duration for the group.
-	The [=Director=] <em class="rfc2119">may</em> decide to close a group
+	The [=Team=] <em class="rfc2119">may</em> decide to close a group
```

Comment:
This one I question. The team could advise, perhaps, but decide?

Reply:
If the team is advising instead of deciding, then we need a process that clarifies who the team is advising so they may make a decision.